dvclive/report.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+import json
+from pathlib import Path
+
+from dvc_render.html import render_html
+from dvc_render.image import ImageRenderer
+from dvc_render.vega import VegaRenderer
+
+from dvclive.data import PLOTS, Image, Scalar
+from dvclive.data.plot import Plot
+from dvclive.utils import parse_tsv, to_base64_url
+
+
+def get_scalar_renderers(scalars_folder):
+    renderers = []
+    for suffix in Scalar.suffixes:
+        for file in Path(scalars_folder).rglob(f"*{suffix}"):
+            data = parse_tsv(file)
+            for row in data:
+                row["rev"] = "workspace"
+            name = str(file.relative_to(scalars_folder).with_suffix(""))
+            properties = {"x": "step", "y": name}
+            renderers.append(VegaRenderer(data, name, **properties))
+    return renderers
+
+
+def get_image_renderers(images_folder):
+    renderers = []
+    for suffix in Image.suffixes:
+        for file in Path(images_folder).rglob(f"*{suffix}"):
+            name = str(file.relative_to(images_folder))
+            data = [
+                {
+                    ImageRenderer.SRC_FIELD: to_base64_url(file),
```

Comment:
So this changes current dvc behavior, right? @pared Do you remember why we decided to include refs to all image files instead of encoding them directly into the HTML?

Reply:
There wasn't any necessity that would make us use `<img>`. We decided to go this way to avoid producing gigantic htmls which would be troubling to parse/inspect.

Reply:
Looking at this again, it seems like it could result in giant htmls if there's enough image data. It also duplicates what's already saved in `dvclive/images`. Can we refer to those image files instead? Is there a reason to prioritize encoding them?

Reply:
> Can we refer to those image files instead? 

Yes, we can.

> Is there a reason to prioritize encoding them?

No strong reason, just making the HTML standalone, similar to how the plot data is embedded. If we refer to the image folder the report might break if someone remove/alter images from the folder.

TBH I think the vega embedding takes proportionally much more space per item than the base64 encoded string for each image.

Reply:
To clarify, the duplication is not introduced in this P.R. 

In the current DVC HTML, image data is being duplicated and saved in a separate directory (https://github.com/iterative/dvc/issues/6665)

Reply:
True, although in this case we already have everything consolidated into a folder. Anyway, we can keep as is for now. No need to block the PR.